test/test_collection.py

```diff
@@ -786,6 +788,26 @@ def test_insert_many(self):
         self.assertFalse(result.acknowledged)
         self.assertEqual(20, db.test.count_documents({}))
 
+    def test_insert_many_invalid(self):
+        db = self.db
+        db.test.drop()
```

Comment:
I see that many tests in this file do this at the top which, I assume is why you included it here. However, there is also the fact that the class is wired to do this before _and_ after every test:

```
    def setUp(self):
        self.db.test.drop()

    def tearDown(self):
        self.db.test.drop()
```

Seems like overkill to me. I think we can remove the drop() from this newly added test at least.

Reply:
ok sounds good!